test/AST-Quake/passQregNToQSpan.cpp

```diff
@@ -29,7 +29,7 @@ struct entry {
 // CHECK-LABEL:   func.func @__nvqpp__mlirgen__entry() attributes {
 // CHECK:           %[[VAL_1:.*]] = quake.alloca !quake.veq<3>
 // CHECK:           %[[VAL_2:.*]] = quake.relax_size %[[VAL_1]] : (!quake.veq<3>) -> !quake.veq<?>
-// CHECK:           call @__nvqpp__mlirgen__function_mcx._Z3mcxN5cudaq5qspan{{.*}}(%[[VAL_2]]) : (!quake.veq<?>) -> ()
+// CHECK:           call @__nvqpp__mlirgen__function_mcx._Z3mcxN5cudaq5{{.*}}(%[[VAL_2]]) : (!quake.veq<?>) -> ()
```

Comment:
```suggestion
// CHECK:           call @__nvqpp__mlirgen__function_mcx._Z3mcxN5cudaq5qview{{.*}}(%[[VAL_2]]) : (!quake.veq<?>) -> ()
```
?

Reply:
I had that, but then I figured why not just slurp the qview up too. Ultimately I thought that keeping qview was just a test that Clang can mangle the function signature so I decided we didn't need to test that here.

Reply:
That's fine. I thought this might be trying to disambiguate the call from something else.